src/test/python_tests/test_formatting.py

```diff
@@ -138,3 +138,127 @@ def test_skipping_site_packages_files():
 
     expected = None
     assert_that(actual, is_(expected))
+
+
+def test_skipping_excluded_files():
+    """Test skipping formatting when the file is in excluded pattern"""
+
+    UNFORMATTED_EXCLUDED_FILE_PATH = constants.TEST_DATA / "sample3" / "sample.unformatted"
+    with session.LspSession() as ls_session:
+        # Use any stdlib path here
+        uri = utils.as_uri(pathlib.__file__)
```

Comment:
The `uri` here should be the path to the file you want to run formatting on. In the previous test case, I was specifically testing skipping stdlib files. Fix the cases below as well.

Reply:
@karthiknadig , how do I debug a test? I added couple of print statements in lsp_server but I am not able to see them during execution.